src/System.Xml.XmlSerializer/src/netcore50/project.json

```diff
@@ -4,6 +4,7 @@
       "dependencies": {
         "Microsoft.NETCore.Targets": "1.0.0",
         "System.Collections": "4.0.10",
+        "System.Collections.NonGeneric": "4.3.0-beta-devapi-24503-01",
```

Comment:
Moreover, why do you even have a netcore50 project.json when this library doesn't have a netcore50 targetgroup in the builds file?